app/src/main/java/com/orange/ouds/app/ui/components/checkbox/CheckboxItemDemoScreen.kt

```diff
@@ -0,0 +1,177 @@
+fun CheckboxItemDemoScreen() = DemoScreen(rememberCheckboxItemDemoState()) {
+    CustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_controlItem_icon_label),
+                checked = icon,
+                onCheckedChange = { icon = it },
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_controlItem_divider_label),
+                checked = divider,
+                onCheckedChange = { divider = it },
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_controlItem_inverted_label),
+                checked = inverted,
+                onCheckedChange = { inverted = it },
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_common_enabled_label),
+                checked = enabled,
+                onCheckedChange = { enabled = it },
+                enabled = enabledSwitchEnabled
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_controlItem_readOnly_label),
+                checked = readOnly,
+                onCheckedChange = { readOnly = it },
+                enabled = readOnlySwitchEnabled
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_common_error_label),
+                checked = error,
+                onCheckedChange = { error = it },
+                enabled = errorSwitchEnabled
+            )
+            CustomizationTextField(
+                label = stringResource(R.string.app_components_common_text_label),
+                value = text,
+                onValueChange = { value -> text = value }
+            )
+            CustomizationTextField(
+                label = stringResource(R.string.app_components_controlItem_helperText_label),
+                value = helperText.orEmpty(),
+                onValueChange = { value -> helperText = value })
+        }
+    ) {
+        CheckboxItemDemo(state = this@DemoScreen)
+        OudsThemeTweak(OudsTheme.Tweak.Invert) {
+            CheckboxItemDemo(state = this@DemoScreen)
+        }
+        CheckboxItemDemoCodeSnippet(
+            state = this@DemoScreen,
+            modifier = Modifier
+                .padding(all = OudsTheme.spaces.fixed.medium)
+                .padding(top = OudsTheme.spaces.fixed.medium)
+        )
+    }
+}
+
+@Composable
+private fun CheckboxItemDemo(state: CheckboxItemDemoState) {
+    with(state) {
+        Box(
+            modifier = Modifier
+                .background(OudsTheme.colorScheme.background.primary)
+                .padding(vertical = OudsTheme.spaces.fixed.medium, horizontal = OudsTheme.grids.margin)
```

Comment:
Other components use `OudsTheme.spaces.fixed.medium` instead of `OudsTheme.grids.margin` as horizontal margins. Maybe we should use `OudsTheme.grids.margin` for the other components too?

Reply:
You're right, I referenced your remark in a new issue as it is not the purpose of this one.